packages/matter.js/src/protocol/interaction/InteractionClient.ts

```diff
@@ -76,19 +77,36 @@ export function ClusterClient<
         attributes,
         events,
         commands,
-        subscribeAllAttributes: async (minIntervalFloorSeconds: number, maxIntervalCeilingSeconds: number) => {
+        subscribeAllAttributes: async (minIntervalFloorSeconds: number, maxIntervalCeilingSeconds: number, keepSubscriptions?: boolean, isFabricFiltered?: boolean) => {
             if (interactionClient === undefined) {
                 throw new InternalError("InteractionClient not set");
             }
-            return await interactionClient.subscribeMultipleAttributes([{ endpointId: endpointId, clusterId: clusterId }], minIntervalFloorSeconds, maxIntervalCeilingSeconds, (data) => {
-                const { path, value } = data;
-                const attributeName = attributeToId[path.attributeId];
-                if (attributeName === undefined) {
-                    logger.warn("Unknown attribute id", path.attributeId);
-                    return;
+            return await interactionClient.subscribeMultipleAttributesAndEvents(
+                [{ endpointId: endpointId, clusterId: clusterId }],
+                [{ endpointId: endpointId, clusterId: clusterId }],
+                minIntervalFloorSeconds,
+                maxIntervalCeilingSeconds,
+                keepSubscriptions,
+                isFabricFiltered,
+                (attributeData) => {
```

Comment:
Don't need the parens (mentioning because I saw you remove them in an another change)

Reply:
right